Yona/Yona/UserRequestManager.swift

```diff
@@ -141,6 +143,27 @@ class UserRequestManager{
         }
     }
     
+    func getUserFetchURL() -> String {
+        return compareAndSwapSchemes(leftURL: getSavedUserFromUserDefaults().getSelfLink!, rightURL: EnvironmentManager.baseUrlString()!,userLinkFromKeychain: KeychainManager.sharedInstance.getUserSelfLink()!)
+    }
+    
+    func compareAndSwapSchemes(leftURL:String, rightURL:String, userLinkFromKeychain:String) -> String {
```

Comment:
Change this into something more meaningful:
```
    func correctUserFetchUrlIfNeeded(userURLStr:String, environmentBaseURLStr:String, storedUserUrlStr:String) -> String {
```
And update the below declarations accordingly.

Reply:
Done.